lib/common/readable-ipc-stream.js

```diff
@@ -0,0 +1,44 @@
+const Readable = require('stream').Readable
```

Comment:
doesn't this file need to be added to the asar output in gn?

Reply:
I don't know enough about the build process to answer to that.  We took the existing `deep-equal` local dependency used by [`extensions.js`](https://github.com/brave/muon/pull/507/files#diff-2f0e53cd5b958243dec767f9b7965fbcR6) as our guide. There's no reference to `deep-equal` in a gn file, so assumed we didn't need to.

Reply:
sorry, I meant `readable-ipc-stream.js`, not the `require('stream'`

Reply:
@bridiver yep, that's what i figured you meant. I was trying to say that we looked at the use site over in https://github.com/tableflip/muon/blob/5c42c5d9755fef62518ae8a883e6ffb9aa8c20c4/lib/browser/api/extensions.js#L3-L6 and followed the pattern that was used for `deep-equal` which we didn't find referenced in a gn file.